gr-utils/python/modtool/modtool_add.py

```diff
@@ -110,7 +110,9 @@ def setup(self, options, args):
         if not options.license_file:
             self._info['copyrightholder'] = options.copyright
             if self._info['copyrightholder'] is None:
-                self._info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+                self._info['copyrightholder'] = raw_input("Please specify the copyright holder: ")
```

Comment:
Can you surround this with

```
with SequenceCompleter(list_of_sensible_candidates):
```

please, so that people can just "<tab>" to get predefined answers?

This would require you to generate a `list_of_sensible_candidates`, for example by `import getpass; getpass.getuser()` (to get the login user), and by asking `git config --global user.name`.

Does that sound feasible?

Reply:
Can you please suggest me the members for the `list_of_sensible_candidates`?
And shall I make it mandatory for the user to add a copyright holder from the same list?